jsonminify/index.d.ts

```diff
@@ -3,6 +3,6 @@
 // Definitions by: Dan Homola <https://github.com/no23reason>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function minify(json: string): string;
+declare const minify: (json: string) => string;
```

Comment:
The problem is that the tests are using `import * as jsonminify from "jsonminify";`, an ES6-style import.
jsonminify is a CommonJS module, so you need to import it using `import jsonminify = require("jsonminify")`.